versions/ufo3/glyphs/glif.md

```diff
@@ -206,6 +206,14 @@ Authoring tools may convert instances of contours like this to anchor elements w
 {: #outline }
 ### outline: Outline description.
 
+Note that this element contains descriptions of graphical data. Therefore, 
+
+1. the order of contours and components may be relevant for other (hinting) data but has no inherent meaning.
+2. the result of component decomposition is implementation-defined.
+3. an empty outline element is semantically equivalent to the element not existing in the first place.
+4. an empty contour element is semantically equivalent to the element not existing in the first place.
+5. a component pointing to an empty glyph is semantically equivalent to the element not existing in the first place.
```

Comment:
Disagree with this: it could be that the glyph just isn't drawn yet, or outlines removed to be redrawn, etc. If you are viewing a UFO as a file representing the finished state, yes, this makes sense. But as a design document were all might not be done yet, no. An equivalent would be an indesign doc that has blank pages, the designer knows those pages will be filled, but they aren't yet.

Reply:
Hm 🤔 Ok, will remove this.